psga/synthetic/generate_synthetic_output.nf

```diff
@@ -20,10 +20,8 @@ process generate_synthetic_output {
   output_csv_file="results.csv"
   output_json_file="resultfiles.json"
 
-  python ${PSGA_ROOT_PATH}/scripts/synthetic/generate_results.py \
+  python ${PSGA_ROOT_PATH}/scripts/synthetic/generate_pipeline_results_files.py \
     --metadata-file "${ch_metadata}" \
-    --output-csv-file "${ch_output_csv_file}" \
-    --output-json-file "${ch_output_json_file}" \
     --output-path "${params.output_path}"
```

Comment:
Same script name as for the sars-cov-2 pipeline so that we keep things as clear as possible.